docs/_redirects

```diff
@@ -24,6 +24,7 @@ https://www.usehardhat.com/* https://hardhat.org/:splat 301!
 /console-log /hardhat-network/#console-log 302
 /discord https://discord.gg/TETZs2KK4k 302
 /hre /advanced/hardhat-runtime-environment.html 302
+/nodejs-versions /reference/stability-guarantees.html 302
```

Comment:
Should we show this in the navbar?

Reply:
If you meant the sidebar, yes. I'll add it.